src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrent.cs

```diff
@@ -28,7 +28,7 @@ public class QBittorrent : TorrentClientBase<QBittorrentSettings>
                            Logger logger)
             : base(torrentFileInfoReader, httpClient, configService, diskProvider, remotePathMappingService, logger)
         {
-            _proxy = proxy;
+            _proxy = proxySelector.GetProxy(Settings);
```

Comment:
Don't call this in the ctor, the Settings won'be populated yet. replace `_proxy.` with `GetProxy(Settings).` in the rest of the class. (Or probably cleaner, create a private `IQbittorrentProxy Proxy => GetProxy(Settings);` helper property)